draft-ietf-ppm-dap-taskprov.md

```diff
@@ -383,6 +383,10 @@ Extensions are treated as mandatory-to-implement in the protocol described in
 {{taskprov}}. In particular, protocols participants MUST opt-out of tasks
 containing unrecognized extensions. See {{provisioning-a-task}}.
 
+If the same extension type appears more than once among the Taskprov extensions
+in the `TaskConfig` structure, then protocols participants MUST opt-out of tasks
+with repeated extensions.
```

Comment:
To have parity with report extension handling, we should treat the entire `TaskConfig` structure as invalid. That would mean aborting DAP rather than just opting out. Likewise below.

Reply:
I think opt-out will have the same effect, because for both Leader and Helper, if they decide to opt-out, the result is abort DAP with invalidTask:
https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap-taskprov/blob/d96bb5a98e424a03d40e2a7608e518c659cbef14/draft-ietf-ppm-dap-taskprov.md?plain=1#L558
https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap-taskprov/blob/d96bb5a98e424a03d40e2a7608e518c659cbef14/draft-ietf-ppm-dap-taskprov.md?plain=1#L607

Reply:
Fair point! But I think this should be handled as `invalidMessage` instead, since we're saying that it's some how "gramatically incorrect" to include the same code point multiple times.

Reply:
It only makes sense for invalidMessage if it's provisioned in-band. But this section describes the overall behavior of a bad taskConfig, it might happen out-of-band, when no Reports have been seen yet.

Reply:
Yes, good point!